src/components/ProposalListGrid.tsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 import { ProposalListGridItem } from './ProposalListGridItem';
 import { DataViewerProposal } from '../interfaces/DataViewerProposal';
 
 interface ProposalListGridProps {
   proposals: DataViewerProposal[];
 }
 
-export const ProposalListGrid = ({ proposals }: ProposalListGridProps) => (
-  <div className="proposal-list-grid">
-    {proposals.map((proposal, index) => (
-      <ProposalListGridItem
-        key={index} // eslint-disable-line react/no-array-index-key
-        proposal={proposal}
-      />
-    ))}
-  </div>
-);
+export const ProposalListGrid = ({ proposals }: ProposalListGridProps) => {
+  const { proposalId } = useParams();
```

Comment:
@jasonaowen Is it okay for a component to reach into params like this? It made sense at the time so I wasn't passing the prop down through all the components from the page, but upon waking up this morning I realized maybe only pages should access params? I also realized I'm out of the loop on what Redux-like global stores React might have these days to get around prop-batons. Anyway, your thoughts here appreciated!

Reply:
Thanks for calling this out! I'd tend to prefer that the current proposal ID be passed in as a prop, primarily because the `<ProposalListGrid>` shouldn't know or care how it's rendered; the fact that this is a route parameter should only be the responsibility of the component responsible for that route. Plus, it'd let us show off the highlighted proposal row in storybook!

Reply:
Two great reasons, I'll add the prop-baton!

Reply:
Done and ready for re-review!

I also added a story as you made reference to in your review, which necessitated one new prerequisite commit so that sample proposals data could be shared between the stories.